q2_phylogenomics/__init__.py

```diff
@@ -0,0 +1,13 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-2019, QIIME 2 development team.
```

Comment:
```suggestion
# Copyright (c) 2020, QIIME 2 development team.
```

Reply:
Don't forget to make that change in all the other copyright headers (and LICENSE).

Reply:
done, thanks!

Reply:
I hate to be "that guy" - but the [LICENSE](https://github.com/qiime2/q2-phylogenomics/blob/a1ab15a04570257fbf161dcb303ff5eef39ba37d/LICENSE#L3) doesn't match, it should read

```
Copyright (c) 2020, QIIME 2 development team.
```

A good way to double-check all of this is to run `q2lint` locally.